src/apis/LocalApi.js

```diff
@@ -1,5 +1,13 @@
-import axios from 'axios';
+import axios from "axios";
+import store from "./../store";
 
-export default axios.create({
-    baseURL: process.env.REACT_APP_BASEURL
-});
\ No newline at end of file
+const LocalApi = axios.create({
+  baseURL: process.env.REACT_APP_BASEURL
+});
+
+LocalApi.interceptors.request.use(config => {
```

Comment:
It doesn't look like this code does anything, but it will help us **a lot** with auth.